src/ui/apply.ui

```diff
@@ -1,103 +1,56 @@
+            <property name="halign">start</property>
           </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">False</property>
-            <property name="position">0</property>
-          </packing>
         </child>
         <child>
-          <object class="GtkBox">
-            <property name="visible">True</property>
-            <property name="can-focus">False</property>
-            <property name="orientation">vertical</property>
-            <child>
-              <object class="GtkLabel" id="dynamic_label">
-                <property name="visible">True</property>
-                <property name="can-focus">False</property>
-                <property name="margin-bottom">20</property>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="position">0</property>
-              </packing>
-            </child>
-            <child>
-              <object class="GtkCheckButton" id="apply_to_queue">
-                <property name="label" translatable="yes">Apply to all queue</property>
-                <property name="visible">True</property>
-                <property name="can-focus">True</property>
-                <property name="receives-default">False</property>
-                <property name="margin-bottom">20</property>
-                <property name="draw-indicator">True</property>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="position">1</property>
-              </packing>
-            </child>
+          <object class="GtkLabel" id="file_list">
+            <property name="halign">start</property>
           </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">True</property>
-            <property name="position">1</property>
-          </packing>
         </child>
       </object>
     </child>
     <action-widgets>
-      <action-widget response="-8">compress_button</action-widget>
-      <action-widget response="-9">skip_button</action-widget>
+      <action-widget response="yes">continue_button</action-widget>
+      <action-widget response="no">cancel_button</action-widget>
     </action-widgets>
+    <child internal-child="action_area">
+      <object class="GtkBox">
+        <property name="margin-start">20</property>
+        <property name="margin-end">20</property>
+        <property name="margin-bottom">20</property>
+        <property name="spacing">10</property>
+        <property name="halign">start</property>
+        <child>
+          <object class="GtkButton" id="continue_button">
+            <property name="label" translatable="1">Continue</property>
```

Comment:
This should be `C_ontinue`, (The C is already taken by `_Cancel`) Maybe one can use `_Ok` instead.